lib/builtIn/ForLoop.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const DefaultConfig = require('../DefaultConfig');
+const DEFAULT_CONFIG = {
+    maxValue: 10,
+    tagName: 'loop',
+    format: '${name} (${i})'
+};
+
+/**
+ * ForLoop to repeat a scenario for a given number of times.
+ * @class
+ * @extends DefaultConfig
+ */
+class ForLoop extends DefaultConfig {
+    //TODO: constructor config scenario outline numberingből
```

Comment:
Could you please remove TODOs if they are not necessary anymore?

Reply:
done